scripts/reborn_webui_v2_live_qa/test_run_live_qa.py

```
@@ -37,6 +39,46 @@ def _dummy_ctx(self) -> run_live_qa.LiveQaContext:
             env={},
         )
 
+    def _fake_assistant_reply_page(self, response_text: str):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_fake_assistant_reply_page`

(ANN202)